apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_info.jsx

```diff
@@ -103,11 +106,21 @@ export class WorkshopEnrollmentSchoolInfo extends React.Component {
     return strs.join(', ');
   }
 
+  // Gets variable containing appropriate list of dropdown options given a course
+  scholarshipDropdownOptions(course) {
+    return CourseSpecificScholarshipDropdownOptions[course];
+  }
+
   scholarshipInfo(enrollment) {
     if (enrollment.scholarship_ineligible_reason) {
       return <td>{enrollment.scholarship_ineligible_reason}</td>;
     }
 
+    let workshopCourseAbbreviation = CourseKeyMap[this.props.workshopCourse];
+    let dropdownOptions = this.scholarshipDropdownOptions(
```

Comment:
nit: since this function is only called once and is one line you could remove it and call CourseSpecificScholarshipDropdownOptions[course] here

Reply:
I think I had some grand vision of splitting things up to make them more readable, but I think you're right. Will update!